File: wallet/src/main/java/com/worldventures/wallet/ui/settings/help/video/delegate/WalletVideoCallback.java
package com.worldventures.wallet.ui.settings.help.video.delegate;

import com.worldventures.core.modules.video.cell.ProgressVideoButtonActions;
import com.worldventures.wallet.ui.settings.help.video.model.WalletVideoModel;

public interface WalletVideoCallback extends ProgressVideoButtonActions<WalletVideoModel> {

   void onPlayVideoClicked(WalletVideoModel entity);
}

File: wallet/src/main/java/com/worldventures/wallet/ui/settings/help/video/delegate/WalletVideoCallback.kt
package com.worldventures.wallet.ui.settings.help.video.delegate

import com.worldventures.core.modules.video.cell.ProgressMediaButtonActions
import com.worldventures.wallet.ui.settings.help.video.model.WalletVideoModel

interface WalletVideoCallback : ProgressMediaButtonActions<WalletVideoModel> {

   fun onPlayVideoClicked(entity: WalletVideoModel)
}
